mern-ecommerce/controllers/auth.js

```diff
@@ -80,3 +80,13 @@ exports.isAdmin = (req, res, next) => {
   }
   next();
 };
+
+exports.checkUser = (req, res, next) => {
```

Comment:
This function seems to be redundant with the `isAuth` function, which has the same functionality. It would be better to remove this function to avoid redundancy.